frontend/Makefile

```diff
@@ -37,6 +37,11 @@ retrieve-vars:
 upload-vars:
 	aws s3 cp .env.production $(S3_ENVIRONMENT_FILE)
 
+.PHONY: retrieve-descendants
+retrieve-descendants:
+	curl -o src/components/common/Filter/descendant_mappings/tissue_descendants.json https://raw.githubusercontent.com/chanzuckerberg/cellxgene-ontology-guide/main/ontology-assets/tissue_descendants.json
```

Comment:
I worry that pulling from main could introduce too much variability. Around the time of schema migration we would need to make sure it's only pulling the tissue descendants from the current schema.